queries/highlights.scm

```diff
@@ -34,12 +34,16 @@
 
```

Comment:
**Lines 9-24: `Function Calls` and `Builtin Functions`**

Not sure why they went for such a change, but these two sections were reordered to have `Builtin functions` to be before `Function calls`. For consistency sake, can we do the same? 

(again, sorry for the misplaced comment)